src/index.js

```diff
@@ -51,12 +56,22 @@ const INSTRUMENTATIONS = [
 
 const NOOP = () => {};
 
+function buildExporter(options) {
+  const completeUrl = options.beaconUrl + (options.rumAuth ? '?auth='+options.rumAuth : '');
+  return options.exporter._factory({
+    beaconUrl: completeUrl,
+    onAttributesSerializing: options.exporter.onAttributesSerializing,
+  });
+}
+
```

Comment:
Do we need it to be a factory? Would simplify it down to

```js
return new SplunkExporter({
    beaconUrl: completeUrl,
    ...options.exporter
})
// or I'd even rather
return new SplunkExporter(completeUrl, options.exporter)
```

Reply:
It definitely should be configurable one way or another. a) feature parity with Nodejs bistro b) less coupling. If you have ideas for a better config format, let me know.